routers/api/v1/repo/repo.go

```diff
@@ -306,6 +306,11 @@ func Migrate(ctx *context.APIContext, form auth.MigrateRepoForm) {
 		return
 	}
 
+	if !ctxUser.IsOrganization() && ctx.User.ID != ctxUser.ID {
```

Comment:
This will disallow an administrator to migrate all users from site A to site B :neutral_face:

Reply:
I would wrap this `if`-statement and the next one inside an `if !ctx.User.IsAdmin`-block to fix that

Reply:
@bkcsoft done